Mergin/validation.py

```diff
@@ -217,6 +219,14 @@ def check_attachment_widget(self):
                         if "UseLink" in cfg:
                             self.issues.append(SingleLayerWarning(lid, Warning.ATTACHMENT_HYPERLINK))
 
+                    # check that expression uses Mergin variables
+                    if "PropertyCollection" in cfg:
+                        if "properties" in cfg["PropertyCollection"]:
+                            if "propertyRootPath" in cfg["PropertyCollection"]["properties"] and "expression" in cfg["PropertyCollection"]["properties"]["propertyRootPath"]:
+                                formula = cfg["PropertyCollection"]["properties"]["propertyRootPath"]["expression"]
```

Comment:
Maybe all this could be rewritten as 
```
try:
  formula = cfg["PropertyCollection"]["properties"]["propertyRootPath"]["expression"]
catch KeyError:
  return
```